project/PlayArtifact.scala

```diff
@@ -41,9 +41,6 @@ object PlayArtifact extends Plugin {
       }
       IO.zip(playArtifactResources.value, distFile)
 
-      // Tells TeamCity to publish the artifact => leave this println in here
```

Comment:
this may be a can of worms, but isn't this covered by the riffraff plugin?

Reply:
It is in later versions IIUC. Given that it's commented out it's not getting executed so should have no impact.